_topic_map.yml

```diff
@@ -392,10 +392,10 @@ Topics:
     File: remove-additional-network
   - Name: Configuring PTP
     File: configuring-ptp
-- Name: OpenShift SDN
+- Name: OpenShift SDN network provider
   Dir: openshift_sdn
   Topics:
-  - Name: About OpenShift SDN
+  - Name: Learn about the OpenShift SDN network provider
```

Comment:
Should it be Learn -> Learning ?

I know it's a concept, so maybe it shouldn't be a gerund, but it just looked off to me. Personal preference I liked "About OpenShift SDN" better.

Same question for the "Learn about the OVN-Kubernetes network provider" assembly

Reply:
@bergerhoffer, I wonder myself.